### ISSUE TITLE
Look at possibly using fish-shell instead of bash

### ISSUE THREAD

**Issue by Jacalz** [FyshOS/fyshos#2]
This would have to be looked at carefully to not scare away people that are used to bash and to make sure that it works reliably. The idea with [fish-shell](https://github.com/fish-shell/fish-shell) is to be friendly and easy to use. I think it fits quite nicely if we want to reinvent the Linux desktop and think a little outside the box.

**Comment by ghost** [comment]
Issue should possibly be closed; As it seems we sided on Zsh?

**Comment by Jacalz** [comment]
Not necessarily. I think there just hasn't been time to evaluate looking into fish. Zsh probably just seemed like a better alternative than Bash.

**Comment by ghost** [comment]
For the sake of posterity, I also did throw [Elvsh](https://elv.sh) on the table in Discord a week or-three back. It's so close to the ideal solution imo, but the custom shell scripting language has me a bit gun-shy from seriously considering it (even though, I find the control flow of it -- from what I've seen, pretty nice. lol).

But yeah, I'm kinda mixed on non-standard adjacent shells in the first-place I guess; Personally think if we were going to do something 'so different' we need to be able to own it in a relatively intimate way --  and I'm not even exactly sure to what extent that even / actually is;  Maybe that's like something on-the-spectrum of it needing to actually be in Go or something -- so there's at least the potential we can make some edits without fully committing to a different ecosystem, but idk.

**Comment by Jacalz** [comment]
I've used fish quite a lot recently and it is a very pleasant experience. Incredibly helpful in command completion.

**Comment by andydotxyz** [comment]
Fish is now the default
